README.txt

```diff
@@ -1,14 +1,14 @@
 INSTALL
 ==========
 
-1. Extract module into /sites/all/modules folder.
+1. Extract module into /modules folder.
 2. Enable module on /admin/modules page.
 
 
 USAGE
 ==========
 
-Go to /admin/config/people/accounts/settings page and use new token [user:password] in mail texts.
+Go to /admin/config/people/accounts page and use new token [user:password] in mail texts.
```

Comment:
Yes. It was changes, as I see http://storage8.static.itmages.com/i/16/0919/h_1474274321_2525140_de22ef0f23.png